### PR TITLE
Moved Data location from Buckets->Reference. Added documentation for upcoming Jurisdictions

### PR DESCRIPTION
**Pull request by jonesphillip**
Moved Data location from Buckets->Reference. Added documentation for upcoming Jurisdictions

### CODE REVIEW

File: content/r2/reference/data-location.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+To access R2 buckets that belong to a jurisdiction from [Workers](/workers/), you'll need to specify the jurisdiction as well as the bucket name as part of your [bindings](/r2/api/workers/workers-api-usage/#3-bind-your-bucket-to-a-worker) in your `wrangler.toml`:
+
+```toml
+[[r2_buckets]]
+bindings = [
+  { binding = "MY_BUCKET", bucket_name = "<YOUR_BUCKET_NAME>", jurisdiction = "<JURISDICTION>" }
+]
+```
+
+For more information on getting started visit to [Use R2 from Workers](/r2/api/workers/workers-api-usage/).
```
Comment (dcpena) [inline content/r2/reference/data-location.md]:
```suggestion
For more information on getting started, refer to [Use R2 from Workers](/r2/api/workers/workers-api-usage/). 
```

File: content/r2/reference/data-location.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select R2.
+2. Select **Create bucket**.
+3. Enter a name for the bucket.
+4. Under **Location**, select **Specify jurisdiction** and choose a jurisdiction from the list.
+5. Select **Create bucket** to complete the bucket creation process.
+
+### Using jurisdictions from Workers
+
+To access R2 buckets that belong to a jurisdiction from [Workers](/workers/), you'll need to specify the jurisdiction as well as the bucket name as part of your [bindings](/r2/api/workers/workers-api-usage/#3-bind-your-bucket-to-a-worker) in your `wrangler.toml`:
```
Comment (dcpena) [inline content/r2/reference/data-location.md]:
```suggestion
To access R2 buckets that belong to a jurisdiction from [Workers](/workers/), you will need to specify the jurisdiction as well as the bucket name as part of your [bindings](/r2/api/workers/workers-api-usage/#3-bind-your-bucket-to-a-worker) in your `wrangler.toml`:
```

File: content/r2/reference/data-location.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+### Limitations
+
+During the beta, the following services will not interact with R2 resources with assigned jurisdictions:
+* [Super Slurper](/r2/data-migration/)
+* [Logpush](/logs/get-started/enable-destinations/r2/)
+
+### Additional considerations
+
+Once an R2 bucket is created, the jurisdiction can't be changed.
```
Comment (dcpena) [inline content/r2/reference/data-location.md]:
```suggestion
Once an R2 bucket is created, the jurisdiction cannot be changed.
```

**Review by dcpena — APPROVED** [review]

**Comment by Naveed9988-9** [comment]
My information submitted name Naveed iftikhar from Quetta Balochistan
Pakistan phone number 923343398851
Address. Faisal town brewery road Quetta Balochistan Pakistan
Postel coad area 87300
Bank details submitted
PK27TMFB0000000029800288
Plsss myself amount given me payment amount My given me

**Review by bthwaites — APPROVED** [review]

**Pull request merged**